### DISCUSSION TITLE
Tmp is filled and not cleared after its filled.

### DISCUSSION THREAD

**Discussion by ristoavila** [blakeblackshear/frigate#13050] [General]
Frigate stops recording when /tmp/cache is full. It also seems its not cleared for some reason even if the video storage has space.

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/53125513-9096-494f-8e86-4e0a67862795">

**Comment by NickM-27** [comment]
You should create a support discussion

**Reply by ristoavila** [reply]
I tought its a bug that cache gets full filled and never cleared.

**Reply by ristoavila** [reply]
Could not reproduce anymore. Has been now running solid for 5 days. Propably caused by migration from 0.13 to 0.14.

**Reply by ristoavila** [reply]
It seems I was wrong and just got this today after 5 days. See logs and screenshot from the support thread. 
https://github.com/blakeblackshear/frigate/discussions/13095#discussioncomment-10411101
